### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by mzhang28**

### CODE REVIEW

**Comment by mzhang28** [comment]
For issue #22

File: tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+# hei
```
Comment (chaosagent) [inline tests/__init__.py]: newline at end of file

File: tests/test_general.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class TestGeneral():
+    def test_sanity(self):
+        assert "sanity level" > 0
```
Comment (chaosagent) [inline tests/test_general.py]: newline at end of file

Also, shiet dat typing th0

File: .coverage [hunk 3]
```diff
@@ -0,0 +1 @@
+!coverage.py: This is a private format, don't read it directly!{"lines": {"/vagrant/tests/__init__.py": [1], "/vagrant/filters.py": [], "/vagrant/views/events.py": [], "/vagrant/views/oauth.py": [], "/vagrant/forms.py": [], "/vagrant/manage.py": [], "/vagrant/models.py": [], "/vagrant/util.py": [], "/vagrant/views/users.py": [], "/vagrant/tests/test_general.py": [1, 2, 3], "/vagrant/config.py": [], "/vagrant/cal.py": [], "/vagrant/constants.py": [], "/vagrant/views/__init__.py": [], "/vagrant/views/base.py": [], "/vagrant/conftest.py": [1]}}
```
Comment (chaosagent) [inline .coverage]: gitignore this

File: .cache/v/cache/lastfailed [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (chaosagent) [inline .cache/v/cache/lastfailed]: gitignore this

**Pull request merged**